test/headless/specs/utilities.js

```diff
@@ -19,8 +21,9 @@ testModule('elementSatisfiesRatio', class extends TestClass {
         window.STATE.satisfied = true;
       })
     })
-    .wait(30)
+    .wait(constants.time.RAF_THRESHOLD * 5)
```

Comment:
So should the time padding go in `assert`? It would be nice to not have arbitrary numbers in the wait amounts.